scala/SlidingWindow/README.md

```diff
@@ -0,0 +1,7 @@
+# KDA Scala Tumbling Window Example
+This a simple streaming application which uses Scala `3.2.0` and Flink's Java DataStream API.
+The application reads data from Kinesis stream, aggregates it using sliding windows and writes results to output Kinesis stream.
+
+### Build
+- Run `sbt assembly` to build an uber jar 
```

Comment:
How about pre-requisites to install `sbt` and anything else required?

Reply:
Agree. Added `sbt` installation step. Thank you.